content/zh/news/releases/1.13.x/announcing-1.13.7/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Istio 1.13.7 发布公告
+linktitle: 1.13.7
+subtitle: 补丁发布
+description: Istio 1.13.7 补丁发布。
+publishdate: 2022-08-01
+release: 1.13.7
+aliases:
+    - /zh/news/announcing-1.13.7
+---
+
+此版本包含针对 [CVE-2022-31045](/zh/news/security/istio-security-2022-005/#cve-2022-31045) 和错误修复以提高稳定性。我们建议用户安装此版本而不是 Istio 1.13.6，因为 Istio 1.13.7 包含上述 CVE 修复。
+这个版本说明描述了 Istio 1.13.6 和 Istio 1.13.7 之间的区别。
+
+其中包括 9 个安全修复程序。如果你在本地使用 Go，我们建议你升级到这个较新的 Go 版本。
```

Comment:
A clause is missing:
FYI, [Go 1.18.4 has been released](https://groups.google.com/g/golang-announce/c/nqrv9fbR0zE),
which includes 9 security fixes. We recommend you to upgrade to this newer Go version if you are using Go locally.